groups/forms.py

```diff
@@ -67,6 +69,10 @@ def __init__(self, *args, **kwargs):
         button_text = 'Subscribe' if to_subscribe else 'Unsubscribe'
         self.helper.layout = Layout(
             FormActions(
-                Submit('submit', button_text),
+                Submit('subscribe-submit', button_text),
             ),
         )
+        self.helper.form_action = reverse_lazy(
+            'discussion-subscribe',
+            kwargs={'pk': discussion.pk},
+        )
```

Comment:
The helper should be defined on the class or the instance, not both. I realise that you're following existing code here. The mix-up must have slipped through a previous review